packages/web/pages/raids.tsx

```diff
@@ -43,547 +172,217 @@ const RaidsPage: React.FC = () => {
           url="https://my.metagame.wtf/raids"
         />
 
-        <Container maxW="7xl" w="100%">
-          <Heading
-            as="h1"
-            fontSize="6xl"
-            fontWeight={600}
-            color="white"
-            fontFamily="mono"
-            mb={[4, 4, 4, 12]}
-            display="flex"
-            flexDir="row"
-            ref={topRef}
-          >
-            State of Raids
-            <Text pl={4} alignSelf="center" fontSize="4xl">
-              🗺️
-            </Text>
-          </Heading>
-
-          <Text
-            mb={4}
-            maxW={{ base: '100%', lg: '3xl' }}
-            fontSize={{ base: 'lg', md: '2xl' }}
-          >
-            This is a map of all the current raids going on inside MetaGame. As
-            such, they represent a chance for you to earn reputation, money
-            &amp; stake in MetaGame.
-          </Text>
-        </Container>
-
-        <Container maxW="7xl" w="100%">
-          <AspectRatio mb={4} ratio={{ base: 3 / 4, md: 9 / 6 }}>
-            <Box
-              as="iframe"
-              title="MetaGame Raids on Miro"
-              src="https://miro.com/app/live-embed/uXjVOmrCvsw=/?moveToViewport=-23399,-15417,32264,27285&embedId=17519047502"
-              allowFullScreen
-            />
-          </AspectRatio>
-        </Container>
-
-        <Container maxW="7xl" w="100%">
-          {/*
-            The flex is used to make two columns/sections
-            - Main Raids
-            - Phase 1
-            When the screen is narrower than the lg breakpoint, the columns are stacked up to make rows instead, with the Phase 1 column/section at the top            
-          */}
+        <Heading
+          as="h1"
+          fontSize="5xl"
+          fontWeight="bold"
+          color="white"
+          fontFamily="mono"
```

Comment:
I know that `mono` is set to Exo 2 family but in future it won't be, so the safest is probably to use `body` or `exo2` until `heading` gets changed (i think there's work underway for this)